### PR TITLE
Update readme to match new endpoint details

### PR DESCRIPTION
**Pull request by Kirosoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
 # dm-borrower-frontend
+
+The Borrower Frontend provides a website for Borrower to view and sign their
```
Comment (danielmarker) [inline README.md]: for "a" borrower

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+## Contents
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Acceptance Tests](#acceptance-tests)
+
+## Usage
+```
+GET     /health                         -- renders standard Gov UK template
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
+```
```
Comment (danielmarker) [inline README.md:18]: Do we need /searchdeed as the actual interactive endpoint

**Comment by danielmarker** [comment]
Couple comments left

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+## Acceptance Tests
+
+### Running the Tests
+
+All of the acceptance tests are contained within the acceptance-tests folder with the feature files under the features folder and the step-definitions under the steps folder.
+
+If you would like to run all of the acceptance tests then navigate into the acceptance-tests folder and run the following command:
+
+'./run_tests.sh'
```
Comment (danielmarker) [inline README.md]: tests not run_tests

File: README.md [hunk 4]
```diff
@@ -1 +1,60 @@
...
+## Contents
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Acceptance Tests](#acceptance-tests)
+
+## Usage
+```
+GET     /health                         -- renders standard Gov UK template
+GET     /searchdeed                     -- borrower views deed landing page
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
```
Comment (endamccormack) [inline README.md]: Long winded, "POST search for deed" or something similar.

File: README.md [hunk 5]
```diff
@@ -1 +1,60 @@
...
+source tests.sh
+```
+
+Run the app
+```
+python run.py runserver --host 0.0.0.0
+```
+> optional ```--port 9000``` where 9000 is the number of a port you can supply to start the server on.
+
+## Acceptance Tests
```
Comment (endamccormack) [inline README.md]: This crap about acceptance tests I have removed in the past, too much detail, a dev just wants to know where the tests are, what language and how to run them. Simplicity! bullet points over paragraphs.

File: README.md [hunk 6]
```diff
@@ -1 +1,60 @@
...
+> default is localhost:9030
+
+Run the unit tests
+```
+source tests.sh
+```
+
+Run the app
+```
+python run.py runserver --host 0.0.0.0
```
Comment (endamccormack) [inline README.md]: host is also optional

File: README.md [hunk 7]
```diff
@@ -1 +1,60 @@
...
+Optional: export variable for deed-api
+```
+export DEED_API_BASE_HOST=http://localhost:8000
+```
+
+> default is localhost:9030
+
+Run the unit tests
+```
+source tests.sh
```
Comment (endamccormack) [inline README.md]: ./tests.sh ?

File: README.md [hunk 8]
```diff
@@ -1 +1,60 @@
...
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
+```
+
+Install the requirements
+```
+pip install -r requirements.txt
+pip install -r requirements_test.txt
+```
+
+Optional: export variable for deed-api
```
Comment (endamccormack) [inline README.md:27]: If its optional I normally add an "> " in front of it

File: README.md (hunk 1 above)
Comment (endamccormack) [inline README.md]: Its not a website its part of a website, its a service

File: README.md [hunk 9]
```diff
@@ -1 +1,60 @@
...
+```
+GET     /health                         -- renders standard Gov UK template
+GET     /searchdeed                     -- borrower views deed landing page
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
+```
+
+Install the requirements
+```
+pip install -r requirements.txt
+pip install -r requirements_test.txt
```
Comment (endamccormack) [inline README.md]: You don't necessarily need requirements_test unless you're running the unit tests, either add a point about how to run the unit tests and include this there or take it out I would say

File: README.md (hunk 7 above)
Comment (endamccormack) [inline README.md]: wait... does that not run the acceptance tests?

Comment (endamccormack) [inline README.md]: I think you're looking for [this](https://github.com/LandRegistry/dm-borrower-frontend/blob/develop/tests.py)

Comment (Kirosoft) [inline README.md]: should be: source test.sh - I will ammend

File: README.md (hunk 1 above)
Comment (Kirosoft) [inline README.md]: yes, agreed

File: README.md (hunk 4 above)
Comment (Kirosoft) [inline README.md]: ok, I think it is useful to note that the response is a html page though

File: README.md (hunk 5 above)
Comment (Kirosoft) [inline README.md]: I left this in for Nigel/Mandy to review/delete etc

File: README.md (hunk 4 above)
Comment (endamccormack) [inline README.md]: Dont they all return HTML pages though? /health and /searchdeed give you html page, its a frontend, you always expect the response to be a page

**Comment by endamccormack** [comment]
You’ve removed getting started from your pull request and you have unit tests in twice

personally I think it should be
- Usage
- Getting Started
  - Install requirements
  - Run the migration (if any)
  - Running the App
  - Running the unit tests
- Acceptance tests

need clear understandable headers structure and a logical flow for it to be easily understood

File: README.md [hunk 10]
```diff
@@ -1 +1,51 @@
...
+## Run the Unit Tests
+
+Install the requirements
+```
+pip install -r requirements_test.txt
+```
+
+Run unit tests and provide coverage report
+
+'source test.sh'
```
Comment (endamccormack) [inline README.md]: `source test.sh` you mean?

Comment (Kirosoft) [inline README.md]: ok, thought this was a markup. Changed to highlight as a command

**Comment by endamccormack** [comment]
:+1:

**Pull request merged**